### PR TITLE
Templates for WIQA

### PR DESCRIPTION
**Pull request by arnaudstiegler**
This dataset (or at least the template tasks) is difficult to correctly phrase. It might make sense to discard the prompts related to the template tasks and only keep the side tasks (but is it worth keeping the dataset at all in this case?)

### CODE REVIEW

**Comment by arnaudstiegler** [comment]
Working on fixing the failing test (due to some local variable not correctly set)

File: templates/wiqa/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+      - {{ question_para_step | join("\n- ")}}
+
+
+      Question:
+
+      {{question_stem}}
+
+
+      How is the supposed perturbation influence the second effect mentioned. Answer
+      by {{"more, less or no effect"}}
```
Comment (VictorSanh) [inline templates/wiqa/templates.yaml:105]:
```suggestion
      How does the supposed perturbation influence the second effect mentioned? Answer
      by {{"more, less or no effect"}}.
```

File: templates/wiqa/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+
+      Is the supposed perturbation:
+
+      - {{"directly impacting a step of the process"}}
+
+      - {{"indirectly impacting a step of the process"}}
+
+      - {{"not impacting any step of the process"}}
+
```
Comment (VictorSanh) [inline templates/wiqa/templates.yaml:133]:
```suggestion
      - {{"not impacting any step of the process"}}
      
      ?
```

File: templates/wiqa/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,193 @@
+dataset: wiqa
+templates:
+  1bc8d95b-0a50-49f4-a46b-bd752929926d: !Template
+    id: 1bc8d95b-0a50-49f4-a46b-bd752929926d
+    jinja: '{% for k in range(1, question_para_step | length) %}
+
+      - {{question_para_step[k]}}
+
+      {% endfor %}
```
Comment (VictorSanh) [inline templates/wiqa/templates.yaml:9]:
```suggestion
    jinja: '- \n- {{ question_para_step | join("\n- ") }}
```

Comment (VictorSanh) [reply templates/wiqa/templates.yaml:9]: same for `remove_first_step_bis`

Comment (VictorSanh) [reply templates/wiqa/templates.yaml:9]: and `remove_last_step` and `remove_last_step_bis`

**Review by VictorSanh — CHANGES_REQUESTED** [review]
looks good to me!
few minor suggestions. please ping me once corrected, will merge then

File: templates/wiqa/templates.yaml (hunk 3 above)
Comment (arnaudstiegler) [reply templates/wiqa/templates.yaml:9]: Those were actually on purpose:
- for `remove_first_step` and `remove_first_step_bis`, the range starts at 1 to remove the first step. Not sure whether there's an easy way to using slicing with jinja
- for `remove_last_step` and `remove_last_step_bis`, the range stops at length-1 to remove the last step. Again not sure whether we can do the slicing directly with jinja

Comment (arnaudstiegler) [reply templates/wiqa/templates.yaml:9]: Actually, my bad, slicing is working. Will update

**Comment by VictorSanh** [comment]
thanks! merging now, please update the spreadsheet

**Pull request merged**